src/outbound.rs

```diff
@@ -263,6 +263,66 @@ impl Outbound {
         })
     }
 
+    pub(crate) fn drain(
+        &self,
+        sids: Vec<usize>,
+        pong_rx: crossbeam_channel::Receiver<bool>,
+    ) -> io::Result<()> {
+        // take lock, which will be held for the duration
+        // of this operation
+        let mut writer = self.writer.lock();
+
+        if writer.is_disconnected() {
+            return Err(Error::new(
+                ErrorKind::NotConnected,
+                "The client is not currently connected to a server",
```

Comment:
Make consistent with below, so maybe "the connection is not currently connected"

Reply:
which one below? the one directly below is only called when the system is in a permanently closed state, rather than in a temporarily disconnected state. Should the system immediately move to closed if it's disconnected, instead of doing a flush?

Reply:
One referenced connection, the other client. Just wanted them consistent.